editor/app/(demo)/r/[slug]/t/[code]/standard.tsx

```
@@ -0,0 +1,181 @@
+          {instagram && <InstagramIcon className="size-4" />}
+        </div>
+      </div>
+      <div className="text-sm text-muted-foreground flex gap-2">
+        <Link href="/support">고객센터</Link>
+        {privacy && <Link href={privacy}>개인정보</Link>}
+        {homepage && <Link href={homepage}>{homepage}</Link>}
+      </div>
+      <hr className="my-4" />
+      {paragraph && (
+        <p className="text-xs text-muted-foreground/60">
+          <span dangerouslySetInnerHTML={{ __html: paragraph }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)